src/Sulu/Bundle/ContentBundle/Resources/public/js/components/content/excerpt/main.js

```diff
@@ -60,6 +60,8 @@ define([], function() {
 
                 this.sandbox.dom.html(this.$el, tpl);
 
+                this.sandbox.dom.removeClass('.preview-update', 'preview-update');
```

Comment:
Ok, so you assume that values from the excerpt are never shown in a visible manner on the website?